src/main/java/com/microsoft/sqlserver/jdbc/SQLServerMetaData.java

```diff
@@ -60,6 +60,23 @@ public SQLServerMetaData(String columnName, int sqlType, int precision, int scal
         this.scale = scale;
     }
 
+    /**
+     * Constructs a SQLServerMetaData with the column name, SQL type, and length.
+     * The precision for string types is length which is used to determine whether to send vmax header to the server.

```

Comment:
why did we mention precision to this constructor? shouldn't it be on the one above?
we should also add javadoc to this method to explain why this constructor was necessary.